src/ode_default_alg.jl

```diff
@@ -50,6 +50,17 @@ function default_algorithm{uType,tType,inplace}(prob::AbstractODEProblem{uType,t
         alg = Rodas4(linsolve=LinSolveFactorize(qrfact!))
       end
     end
+  else # :auto ∈ alg_hints
+    if uEltype <: AbstractFloat
+      if !(uEltype <: Float64) || tol_level == :extreme_tol
+        # Most likely higher precision, so use a higher order method
+        alg = AutoVern9(Rodas4())
```

Comment:
Why 4?

Reply:
Because `Rodas5` doesn't support `BigFloat`. Should I make a branch in here?